_topic_map.yml

```diff
@@ -3057,6 +3057,8 @@ Topics:
     File: serverless-developing-quarkus-functions
   - Name: Using functions with Knative Eventing
     File: serverless-functions-eventing
+  - Name: Accessing secrets and ConfigMaps from Serverless functions
```

Comment:
I think it's actually "config maps" according to the [OCP guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#general-references)